### ISSUE TITLE
TYPES BEGIN END with one INCLUDE TYPE

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#610]
should give error,

```abap
    TYPES: BEGIN OF dummy1.
        INCLUDE TYPE dselc.
    TYPES: END OF dummy1,
```
